src/components/__tests__/i18n.test.tsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+
+import {render, screen} from '../../../test-utils/utils';
+import {Dialog} from '../Dialog';
+import {Pagination} from '../Pagination';
+import {Lang, configure, registerCustomKeysets} from '../utils/configure';
+
+test('should render components with custom keysets', () => {
```

Comment:
Is it ok, that I don't have `describe`, or should I encapsulate everything in it? I haven't done it, because jest splits tests by files anyway